packages/victory-zoom-container/src/index.d.ts

```diff
@@ -10,20 +10,23 @@
 //                 Alec Flett <https://github.com/alecf>
 
 import * as React from "react";
-import { DomainPropType, VictoryContainerProps, CursorData } from "victory-core";
+import { RangeTuple, VictoryContainerProps, CursorData } from "victory-core";
 
 export interface VictoryZoomContainerProps extends VictoryContainerProps {
   allowPan?: boolean;
   allowZoom?: boolean;
-  clipContainerComponent?: React.ReactElement;
-  zoomDimension?: "x" | "y";
-  zoomDomain?: DomainPropType;
   brushStyle?: React.CSSProperties;
+  clipContainerComponent?: React.ReactElement;
   defaultBrushArea?: "all" | "none" | "disable";
```

Comment:
`defaultBrushArea` should include the option "move"

Reply:
I currently auditing `Zoom-Container` on a new branch. Would you like me to audit the `Zoom-container` on this branch as well? I figured I break it up in different PR. Also I don't see `defaultBrushArea` as an optional prop in the [docs](https://formidable.com/open-source/victory/docs/victory-zoom-container) nor was it included [here](https://github.com/FormidableLabs/victory/blob/master/packages/victory-zoom-container/src/victory-zoom-container.js#L18-L35).